app/src/main/java/ch/epfl/balelecbud/utility/storage/FirebaseStorage.java

```diff
@@ -18,8 +18,6 @@
  */
 public final class FirebaseStorage implements Storage {
 
-    private static final String TAG = FirebaseStorage.class.getSimpleName();
-
     private static final FirebaseStorage instance = new FirebaseStorage();
     private final com.google.firebase.storage.FirebaseStorage firebaseStorage = com.google.firebase.storage.FirebaseStorage.getInstance();
```

Comment:
Since we only use it with a call to `getReference()`, couldn't we directly have the reference as a field instead of the `FirebaseStorage` ?